server/routes/v1/version.ts

```diff
@@ -337,6 +341,17 @@ export const deleteVersion = [
 
     await version.delete(user._id)
 
+    // Send email to owners of affected deployments
+    const model = await ModelModel.findById(version.model)
+    if (model) {
```

Comment:
I would think that we should expect to find the model 100% of the time so it would be a valid reason to throw an error if we don't:
```
if (!model) {
  throw NotFound({ code: 'model_not_found', version.model }, `Unable to find model '${version.model}'`)
}

const deployments = await findDeploymentsByModel(user, model)
if (deployments.length > 0) {
  emailDeploymentOwnersOnVersionDeletion(deployments, version)
}
```